Makefile

```diff
@@ -1,22 +1,15 @@
-MAKEFLAGS += --silent
-
-
 .PHONY: help install run stop test
 
+.DEFAULT_GOAL := help
+
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 composer-install: ## Run composer install within the host
 	docker-compose run --no-deps --rm \
-		service_php bash -ci 'bin/composer install'
+		php bash -ci 'bin/composer install'
```

Comment:
I renamed the name of the services, `service_` seems redundant to me. But it's a matter of taste.